crypto/evp/evp_rand.c

```diff
@@ -0,0 +1,532 @@
+            fnrandcnt++;
+            break;
+        case OSSL_FUNC_RAND_GENERATE:
+            if (rand->generate != NULL)
+                break;
+            rand->generate = OSSL_get_OP_rand_generate(fns);
+            fnrandcnt++;
+            break;
+        case OSSL_FUNC_RAND_RESEED:
+            if (rand->reseed != NULL)
+                break;
+            rand->reseed = OSSL_get_OP_rand_reseed(fns);
+            break;
+        case OSSL_FUNC_RAND_NONCE:
+            if (rand->nonce != NULL)
+                break;
+            rand->nonce = OSSL_get_OP_rand_nonce(fns);
+            break;
+        case OSSL_FUNC_RAND_SET_CALLBACKS:
+            if (rand->set_callbacks != NULL)
+                break;
+            rand->set_callbacks = OSSL_get_OP_rand_set_callbacks(fns);
+            break;
+        case OSSL_FUNC_RAND_ENABLE_LOCKING:
+            if (rand->enable_prov_locking != NULL)
+                break;
+            rand->enable_prov_locking = OSSL_get_OP_rand_enable_locking(fns);
+            break;
+        case OSSL_FUNC_RAND_LOCK:
+            if (rand->prov_lock != NULL)
+                break;
+            rand->prov_lock = OSSL_get_OP_rand_lock(fns);
+            break;
+        case OSSL_FUNC_RAND_UNLOCK:
+            if (rand->prov_unlock != NULL)
+                break;
+            rand->prov_unlock = OSSL_get_OP_rand_unlock(fns);
+            break;
+        case OSSL_FUNC_RAND_GETTABLE_PARAMS:
+            if (rand->gettable_params != NULL)
+                break;
+            rand->gettable_params =
+                OSSL_get_OP_rand_gettable_params(fns);
+            break;
+        case OSSL_FUNC_RAND_GETTABLE_CTX_PARAMS:
+            if (rand->gettable_ctx_params != NULL)
+                break;
+            rand->gettable_ctx_params =
+                OSSL_get_OP_rand_gettable_ctx_params(fns);
+            break;
+        case OSSL_FUNC_RAND_SETTABLE_CTX_PARAMS:
+            if (rand->settable_ctx_params != NULL)
+                break;
+            rand->settable_ctx_params =
+                OSSL_get_OP_rand_settable_ctx_params(fns);
+            break;
+        case OSSL_FUNC_RAND_GET_PARAMS:
+            if (rand->get_params != NULL)
+                break;
+            rand->get_params = OSSL_get_OP_rand_get_params(fns);
+            break;
+        case OSSL_FUNC_RAND_GET_CTX_PARAMS:
+            if (rand->get_ctx_params != NULL)
+                break;
+            rand->get_ctx_params = OSSL_get_OP_rand_get_ctx_params(fns);
+            break;
+        case OSSL_FUNC_RAND_SET_CTX_PARAMS:
+            if (rand->set_ctx_params != NULL)
+                break;
+            rand->set_ctx_params = OSSL_get_OP_rand_set_ctx_params(fns);
+            break;
+        case OSSL_FUNC_RAND_VERIFY_ZEROIZATION:
+            if (rand->verify_zeroization != NULL)
+                break;
+            rand->verify_zeroization = OSSL_get_OP_rand_verify_zeroization(fns);
+#ifdef FIPS_MODULE
+            fnfipscnt++;
```

Comment:
has_zeroization or similar is a better name for this variable.

Reply:
Changed to fnzeroizecnt.